MovieQuiz/Models/GameRecord.swift

```diff
@@ -0,0 +1,11 @@
+import Foundation
+
+struct GameRecord: Codable {
+    let correct: Int
+    let total: Int
+    let date: Date
+    
+    func checkingCurrentRecord (game: GameRecord) -> Bool {
```

Comment:
Вот тут пробел не нужен
```suggestion
    func checkingCurrentRecord(game: GameRecord) -> Bool {
```